src/Roles.js

```diff
@@ -74,7 +74,6 @@ const ROLES = {
   PARAGRAPH: 'paragraph',
   REGION: 'region',
   ROW: 'row',
-  ROWGROUP: 'rowgroup',
```

Comment:
This should have been removed when the table body, foot, and head role entries were added since those serve the purpose of this ARIA role while keeping the 1-to-1 role to tag mapping that CAM uses.